src/components/Search/SelectedRevisionsTable/SelectedRevisionsTable.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+
+import Close from '@mui/icons-material/Close';
+import { TableRow } from '@mui/material';
+import IconButton from '@mui/material/IconButton';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import { connect, useDispatch } from 'react-redux';
+
+import { deleteRevision } from '../../../reducers/SelectedRevisions';
+import { Alignment } from '../../../types/enums';
+import { Revision } from '../../../types/state';
+import StyledTableCell from '../../Shared/StyledTableCell';
+import StyledTableRow from '../../Shared/StyledTableRow';
+import './SelectedRevisionsTable.css';
+
+interface TableHeaderDetails {
+  title: string;
+  alignment: Alignment;
+}
+
+const tableHeaderDetails: TableHeaderDetails[] = [
+  { title: 'Project', alignment: 'left' },
+  { title: 'Revision', alignment: 'left' },
+  { title: 'Author', alignment: 'left' },
+  { title: 'Commit message', alignment: 'center' },
+  { title: 'Timestamp', alignment: 'left' },
+];
```

Comment:
In a followup PR, could you move the interface and array into the shared folder also?